users-service/service.yml

```diff
@@ -26,4 +26,7 @@ messages:
 
 
 dependencies:
-  - 'mongo'
+  mongo:
+    prod: 'ds143588.mlab.com:43588'
+    dev: 'mongo:27017'
+    test: 'localhost:27017'
```

Comment:
We are using dedicated MongoDB docker images here. The way it works is that the `27017` port of the running mongo docker image is published to `localhost` seeing as we aren't using the docker `link`s. So we address it as `localhost:27017` but that is actually routed to the mongoDB docker image that is running.